metadata.yaml

```diff
@@ -1,27 +1,33 @@
-contact: Add your name here
-email: Add your email address here
-created: Add the date you started the experiment here (YYYY-MM-DD)
+contact: Andrew Kiss
```

Comment:
I don't think we want to change the contents of this file, other than the `description` and `keywords`